cpp/oneapi/dal/table/backend/homogen_table_builder_impl.hpp

```diff
@@ -227,11 +235,23 @@ class homogen_table_builder_impl {
         }
     }
 
+    void allocate_ftypes_if_need() {
```

Comment:
Why you created a separate method to allocate `ftypes_` but do not allocating it in `allocate` method?